scripts/redis/scan_redis.py

```diff
@@ -1,15 +1,13 @@
+#!/usr/bin/env python
 """Safely scan entire Redis instance and report key stats."""
```

Comment:
Rename the script to something like `redis-bulk-ttl` and document its capabilities?